backend/utils.js

```diff
@@ -0,0 +1,14 @@
+async function logout () {
+    const backendUrlAddress = import.meta.env.VITE_BACKEND_ADDRESS
```

Comment:
Should this file be at this path? This looks like a util function for the frontend, as opposed to the backend

Reply:
You're right. I aded it to my backend folder at first because I initially planned on using it in my backend. Will fix now!